### ISSUE TITLE
Make the getSigFile method in BinarySignatureIdentifier public

### ISSUE THREAD

**Issue by JSLair** [digital-preservation/droid#199]
Could you make the getSigFile method in BinarySignatureIdentifier public, cause I need it to order the results with priorities?

**Comment by anjackson** [comment]
+1 that would be great. Currently I work around this by adding a class in the same package that exposes the method (see [here](https://github.com/openpreserve/nanite/blob/master/nanite-core/src/main/java/uk/gov/nationalarchives/droid/core/DroidSigUtils.java])) but I'd rather be able to do this cleanly.

**Comment by alexgreenDP** [comment]
Yes this should be possible. We will add it to our next release.
